railties/lib/rails/commands/task_builder/test.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'common'
+require 'rake'
+
+module Rails
+  module Commands
+    module TaskBuilder
+      # This is a wrapper around all Rails test tasks, including:
+      #   rails test
+      #   rails test:db
+      class Test
```

Comment:
@kaspth Base was probably a bad name. It's really a wrapper for all existing rails commands. Perhaps I'll rename "Base" to "Common" and "Common" to "HelperMethods".